components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5571,6 +5571,11 @@ private boolean isComplexAttribute(String name) {
         }
     }
 
+    private boolean isBooleanAttribute(String name) {
+
+        return "active".equals(name);
```

Comment:
why were are hardcoding the active name here ?
we should check this from the scim2 configuration file and decide whether it is a bool right ?

Reply:
We have used a similar hardcoded approach for all the methods such as isComplex, isMultiValued etc. here : https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/blob/87d9c7763f64da2ce22ce3689525a32c58ca5e46/components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java#L5554

Therefore, added this method following the same approach.